docs/sources/configuration/integrations/cloudwatch-exporter-config.md

```diff
@@ -0,0 +1,400 @@
+the AWS role that the credentials configured in the environment posses will be used. 
+
+This is useful when scraping metrics from different AWS accounts with a single pair of credentials. In this case, a different role
+is configured for the agent to assume prior to calling AWS APIs, therefore, the credentials configured in the system need
+permission to assume the target role. See [this documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_permissions-to-switch.html) on how to configure this.
+
+```yaml
+  # Required: AWS IAM Role ARN the exporter should assume to perform AWS API calls.
+  role_arn: <string>
+
+  # Optional: External ID used when calling STS AssumeRole API. See https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html for details.
+  external_id: <string>
+```
+
+### aws_dimension
+
+Represents an [AWS CloudWatch Dimension](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Dimension).
+
+```yaml
+  name: <string>
+  value: <string>
+```
+
+### aws_tag
+
+Represents an [AWS Tag](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html).
+
+```yaml
+  key: <string>
+  value: <string>
+```
+
+### metric
+
+Represents an AWS Metrics to scrape, under the context of a job. To see available metrics, AWS does not keep a documentation page with all available metrics.
+Follow [this guide](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/viewing_metrics_with_cloudwatch.html) on how to explore metrics, to easily
+pick the ones you need.
+
+```yaml
+  # Required: CloudWatch metric name.
+  name: <string>
+  
+  # Required: List of statistic types, e.g. "Minimum", "Maximum", etc.
+  statistics: [ <string> ]
+  
+  # Optional: See the `Period` section below.
+  period: [ <duration> | default = 5m ]
+```
+
+### Period
+
+Period controls how far back in time CloudWatch metrics are considered, during each agent scrape. We can split how these 
+settings affects the produced values in two different scenarios.
+
+If all metrics within a job (discovery or static) have the same `Period` value configured, CloudWatch APIs will be requested
+for metrics from the scrape time, to `Periods` seconds in the past. The values of these are exported to Prometheus.
+
+![](../assets/single-period-time-model.png)
+
+On the other hand, if metrics with different `Periods` are configured under an individual job, this works differently.
+First, two variables are calculated aggregating all periods: `length`, taking the maximum value of all periods, and 
+the new `period` value, taking the minimum of all periods. Then, CloudWatch APIs will be requested for metrics from
+`now - length` to `now`, aggregating each in samples for `period` seconds. For each metrics, the most recent sample
+is exported to CloudWatch.
+
+![](../assets/multiple-period-time-model.png)
```

Comment:
This image has .png.png as suffix. Should rename image file. 

There's also a new method for storing images on S3 described in the Writers' Toolkit: https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/#where-to-store-media-assets.